.travis.yml

```diff
@@ -1,12 +1,43 @@
-sudo: false
+install: true
+
+jdk:
+ - openjdk8
+
+# due to absence of rt.jar and others, code broken for Java >= 9 
+# - openjdk11
+
+
 language: java
-jdk: openjdk8
+
+before_script:
+ - echo $JAVA_OPTS
+ - export JAVA_OPTS='-Xms1g -Xmx3g'
+ - echo "MAVEN_OPTS='-Xmx3g'" > ~/.mavenrc
+
+env:
+  global:
+    - MAVEN_OPTS="-Xmx3g"
+
 cache:
     apt: true
     directories:
         - $HOME/.m2
+
 services:
-    - xvfb
-script: mvn clean verify -Pjacoco coveralls:report
-notifications:
-    slack: cunyponder:A8GuEoj0voEn2H7MJDL1eNji
+ - xvfb
+    
+script:
+ - cd edu.cuny.hunter.streamrefactoring.parent && mvn -e generate-sources install && cd -  
+ - du -sh edu.cuny.hunter.streamrefactoring.updatesite/target/repository
```

Comment:
I also saw this in the GH actions build ...

Reply:
the du is just because sometimes upload might fail a bit silently if the files are too large, it also makes sure it's about the right size (some MB) so it was (maybe) built correctly

Reply:
This travis file is obsolete, they stopped offering free service for open source projects so I now only use GH actions. So this file can be deleted it is overridden by the Gh workflow file.